crates/polars-core/src/series/proptest.rs

```diff
@@ -84,8 +84,8 @@ impl Default for SeriesArbitraryOptions {
         Self {
             allowed_dtypes: SeriesArbitrarySelection::all(),
             max_nesting_level: 3,
-            series_length_range: 0..=5,
-            categories_range: 0..=3,
+            series_length_range: 1..=5,
+            categories_range: 1..=3,
```

Comment:
Why did you change this default config to 1? I think the empty input is always a very valuable corner case to test. This also goes for the new `DataFrameArbitraryOptions`.